README.md

```diff
@@ -63,12 +63,22 @@ paymentOptions.setPaymentType(PaymentType.ALL);
 paymentOptions.setStoredCardList(paymentMethodItems);
 
 //set merchant customizations such as page title, icon, and footer text (html)
-paymentOptions.setMerchantTitle("Lucy's Shop");
-paymentOptions.setMerchantIcon(R.drawable.your_icon);
+paymentOptions.setMerchantTitle("Lucy's Shop"); // NOT AVAILABLE!
+paymentOptions.setMerchantIcon(R.drawable.your_icon); // NOT AVAILABLE!
 paymentOptions.setMerchantText("<div style=\"text-align: center;\">\n" +  "Pass in your customized merchant text.</div>");
 
 //set billing address
-paymentOptions.setBillingAddress(new Address("Street 1", "Street 2", "City", "State", "Zip", "Country", null));
```

Comment:
Likewise, `setBillingAddress` is not available, I was able to workaround it using `new CreditCardInfo` and passing that in instead.

Reply:
👍